src/tests/integration/mapping.test.ts

```diff
@@ -22,25 +22,22 @@ describe('/api', () => {
     const subDomain = `testing${uuidv4()}`
     const domain = 'Rahul'
     const port = '5678'
-    await fetch(`${apiURL}/api/mappings`, {
-      method: 'POST',
-      headers: {
-        authorization: ADMIN,
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({
-        domain,
-        subDomain,
-        port,
-      }),
+    const postResponse = await mappingAdapter('/', 'POST', {
+      domain,
+      subDomain,
+      port,
     })
```

Comment:
For thhe future, you can diectly do:
```
const postMapping = await mappingAdapte(....).thhen(r => r.json())
```

Cuently you are not using `postResponse`, so it doesn't have to be there.